build.gradle.kts

```diff
@@ -55,6 +56,9 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter:$junitVersion")
     testImplementation("org.hamcrest:hamcrest:2.1")
 
+    // Mockito
+    testCompile("org.mockito:mockito-core:2.+")
```

Comment:
it should be `testImplementation` according to modern Gradle recommendations and lets fix the version to https://mvnrepository.com/artifact/org.mockito/mockito-core/3.0.0

Reply:
Okay, thanks - I relied on https://site.mockito.org/ which seems outdated